about/index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+
+    <h1>Почему я выбрала Front-end направление.</h1>
+
+<p>Доброго времени суток! Меня зовут Юлия Коленко и я выбрала Front-end разработку, потому, что для меня <b>важно</b>, <br/>как выглядит внешний функционал приложения, с которым контактирует пользователь и интересен сам <i>процесс его создания</i>!</p>
+
+<h3><span style="background: rgb(45, 218, 203);">Мои сильные качества</span></h3>
+<ol>
+    <li>Умение работать как в комаде так и в одиночку</li>
+    <li>Усидчивость, желание всё доводить до конца</li>
+    <li>Творческие способности</li>
+    <li>Ответственность</li>
+    <li>Быстрое переключение между задачами</li>
+    <li>Умение читать и разбираться в чужом коде</li>
+    <li>Самообучение</li>
+</ol>
+
+<hr style="background: rgb(235, 151, 247);">
+
+<p>Однажды Дэн Браун <a href="http://itmydream.com/citati/book/den-braun-cifrovaya-krepost" target="_blank"><mark>сказал</mark></a>: <blockquote>"Возможно всё, на невозможное просто требуется больше времени."</blockquote></p>
+<strong>Когда-то</strong>, и я считала программирование чем-то безумно сложным, <del>невозможным и недоступным</del> для меня.<br/>Сейчас же я понимаю, что нужно просто начать что-то делать, для его понимания, уделять этому <ins>больше времени</ins>, и только тогда <em><b>всё получится!</b></em>
+
+    <hr style="background: rgb(235, 151, 247);">
+
+<h3>Я не знала, что придумать для использования некоторых других тегов,<br/> поэтому я буду решать <del>сложнейшие</del> уравнения из высшей математики и музыкальной сферы.</h3>
+<p>&#128071;&#128071;&#128071;</p>
+
+&#10112; 1 метр<sup>2</sup> = 100<sup>2</sup> дециметров;
```

Comment:
Text can't be without an inline tag.

Reply:
Yes, I forgot. Wrapped it in a paragraph.